uw_hrp/__init__.py

```diff
@@ -4,20 +4,20 @@
 """
 
 import logging
-import json
 from uw_hrp.dao import HRP_DAO
 from restclients_core.exceptions import DataFailureException
 
 
 logger = logging.getLogger(__name__)
+hrp_dao = HRP_DAO()
```

Comment:
We should stop using the pattern, since it causes non-django scripts to crash.

Reply:
Updated. Thank you!

Reply:
Hold,  i'd like to  revise the code